csvimport.json

```diff
@@ -0,0 +1,215 @@
+                    "required": true,
+                    "order": 2
+                },
+                "artifact_name": {
+                    "description": "Name for the artifact (e.g. IP Artifact)",
+                    "data_type": "string",
+                    "required": true,
+                    "order": 3
+                },
+                "artifact_label": {
+                    "description": "Label for the artifact (e.g. IP_test)",
+                    "data_type": "string",
+                    "required": false,
+                    "order": 4
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "Status",
+                    "column_order": 2,
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.artifact_label",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.parameter.artifact_name",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.parameter.cef_column_headers",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.parameter.container_id",
+                    "data_type": "numeric",
+                    "column_name": "Container ID",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.parameter.vault_id",
+                    "data_type": "string",
+                    "column_name": "Vault ID",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.data",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "csv from artifacts",
+            "description": "Given a container_id create a csv in the vault from all the container's artifacts",
```

Comment:
Please update it with more user friendly sentence

Reply:
please provide the sentence which u think would be much user friendly

Reply:
You can update based on the actual functionality of action